it/uk/gov/hmrc/agentclientrelationships/stubs/DataStreamStub.scala

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmrc.agentclientrelationships.audit.AgentClientRelationshipEvent.A
 
 trait DataStreamStub extends Eventually {
 
-  private implicit val patience = PatienceConfig(scaled(Span(15, Seconds)), scaled(Span(500, Millis)))
+  private implicit val patience = PatienceConfig(scaled(Span(25, Seconds)), scaled(Span(500, Millis)))
```

Comment:
10 seconds more? we might want to take another look at this and see if it's a spec in particular